flink-connector-aws/flink-sql-connector-sqs/src/test/java/org/apache/flink/connector/sqs/PackagingITCase.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.flink.connector.sqs;
+
+import org.apache.flink.packaging.PackagingTestUtils;
+import org.apache.flink.table.factories.Factory;
+import org.apache.flink.test.resources.ResourceTestUtils;
+
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+
+/** Packaging test for the SQS SQL connector. */
+public class PackagingITCase {
+
+    @Test
+    void testPackaging() throws Exception {
+        final Path jar = ResourceTestUtils.getResource(".*/flink-sql-connector-sqs[^/]*\\.jar");
+
+        PackagingTestUtils.assertJarContainsOnlyFilesMatching(
+                jar,
+                Arrays.asList(
+                        "org/apache/flink/",
+                        "org/apache/commons/",
```

Comment:
Should we shade and relocate this to `org.apache.flink.connector.sqs.shaded` like the other dependencies?